stdlib/multiprocessing/sharedctypes.pyi

```diff
@@ -92,15 +100,15 @@ class Synchronized(SynchronizedBase[_SimpleCData[_T]], Generic[_T]):
 class SynchronizedArray(SynchronizedBase[ctypes.Array[_CT]], Generic[_CT]):
     def __len__(self) -> int: ...
     @overload
-    def __getitem__(self, i: slice) -> list[_CT]: ...
+    def __getitem__(self, i: slice) -> Any: ...
     @overload
-    def __getitem__(self, i: int) -> _CT: ...
+    def __getitem__(self, i: int) -> Any: ...
     @overload
-    def __setitem__(self, i: slice, value: Iterable[_CT]) -> None: ...
+    def __setitem__(self, i: slice, value: Any) -> None: ...
     @overload
-    def __setitem__(self, i: int, value: _CT) -> None: ...
-    def __getslice__(self, start: int, stop: int) -> list[_CT]: ...
-    def __setslice__(self, start: int, stop: int, values: Iterable[_CT]) -> None: ...
+    def __setitem__(self, i: int, value: Any) -> None: ...
+    def __getslice__(self, start: int, stop: int) -> Any: ...
+    def __setslice__(self, start: int, stop: int, values: Any) -> None: ...
```

Comment:
> As SynchronizedString must be a Subclass of SynchronizedArray the typings must match.

Not necessarily. It's fine to `# type: ignore` overridden methods that break the LSP at runtime. So if `SynchronizedString` *always* slices to `bytes`, the stub should reflect that and use `# type: ignore` if necessary. If on the other hand `SynchronizedString` can be used like `SynchronizedArray` and slice to `list[int]` under certain circumstances, `Any` is probably the best we can do.

Reply:
Uh, thanks! I did not know about this and was hesitant after reading
> Note that in cases where you ignore the overlapping overload error, mypy will usually still infer the types you expect at callsites.
on https://mypy.readthedocs.io/en/stable/more_types.html

But this now looks like it would be working, thanks :)